README.md

```diff
@@ -80,53 +80,94 @@ The full list of what is run for TestGrid is in the [sig-windows-config.yaml](ht
 
 Kubetest is a wrapper that includes everything needed to deploy a cluster, test it (using e2e.test), gather logs, then upload the results to a Google Storage account. It has built-in cloud provider scripts to build Linux+Windows clusters using Azure and GCP.
 
+#### Build kubetest
 
-#### Azure
+Refer to the [kubetest documentation](https://github.com/kubernetes/test-infra/tree/master/kubetest) for full details.
 
-> TODO: This section is still under construction
+```
+git clone https://github.com/kubernetes/test-infra.git
+cd test-infra
+GO111MODULE=on go install ./kubetest
+```
 
-Set environment variables:
+#### Azure
 
-`AZURE_SSH_PUBLIC_KEY_FILE` - Path to the SSH public key you want to use for connecting to the cluster nodes. This is probably `~/.ssh/id_rsa.pub`
+##### Pre-requisites
 
-`AZURE_CREDENTIALS` - Path to a TOML file with a service account credential that will be used for creating the Azure resources
+- Link to AKS engine [release tar file](https://github.com/Azure/aks-engine/releases) or clone aks-engine and build your own using `make dist` then upload to public location.
+- Container registery (ACR or dockerhub). See the [dockerlogin code](https://github.com/kubernetes/test-infra/blob/dd6a466605560e9cbe9a4a2975673cf61dfc7c59/kubetest/aksengine.go#L794) for how it works.
+- [Service Principal](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest)
+- Clone [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) 
+
+## Configuration
+Create a toml file with Azure Authorization information:
 
 ```toml
 [Creds]
   ClientID = ""
   ClientSecret = ""
-  SubscriptionId = ""
+  SubscriptionID = ""
   TenantID = ""
   StorageAccountName = ""
   StorageAccountKey = ""
 ```
 
-Once those are set, you can run `kubetest` and it will do the rest. The full set of tests will take 6-7 hours.
+Set the following environment variables:
+
+```
+# used for ssh to machines 
+export K8S_SSH_PUBLIC_KEY_PATH="/home/user/.ssh/id_rsa.pub"
+
+# used during log collection
+export K8S_SSH_PRIVATE_KEY_PATH="/home/user/.ssh/id_rsa"
+
+# file path to the toml with the auth values
+export AZURE_CREDENTIALS="/home/user/azure/azure.toml"
+
+# location logs will be dumped
+export ARTIFACTS="/home/user/out/kubetest"
+
+# docker registry used 
+export REGISTRY="yourregistry.azurecr.io"
+
+# File required for windows.
+export WIN_BUILD="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/build/build-windows-k8s.sh"
+export KUBE_TEST_REPO_LIST_DOWNLOAD_LOCATION="https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/images/image-repo-list"
```

Comment:
I think `KUBE_TEST_REPO_LIST` needs to point to a local file of `https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/images/image-repo-list`

Reply:
When using kubetest it is managed by the tool it's self if you specify a download location: https://github.com/kubernetes/test-infra/blob/65e48d849253baab70b8f17b1d1b29448761ae50/kubetest/aksengine.go#L1262